CMakeLists.txt

```diff
@@ -43,6 +43,13 @@ option(USE_SYSTEMD "" OFF)
 option(ENABLE_XDGAUTOSTART "" OFF)
 option(ENABLE_EMOJI "" OFF)
 option(ENABLE_LIBUUID "" OFF)
+option(ENABLE_ASAN "Enable Address Sanitizer" OFF)
+option(ENABLE_CODESIGN "Sign the executable file" OFF)
+set(CODESIGN_KEY "" CACHE STRING "The key used for code signing")
```

Comment:
> Is it expected that notifications stay silently in notification center and not pop-up?

No. It should pop up. Did you enable "Do Not Disturb" or "Focus Mode"? If so, it will be hidden in the center.

> rime's ascii mode

Toggling `ascii_mode` won't send notifications. 

> half shape

The timeout for these tips is only 1 second, so it probably disappeared before you see it?